conda/environment.yml

```diff
@@ -16,6 +16,7 @@ dependencies:
   - jupyter-resource-usage
   - mamba
   - pip
+  - python=3.9
```

Comment:
Any reason not to go to 3.10?

Reply:
For reference, the initial work around conda #48  had this at 3.10

Reply:
> Any reason not to go to 3.10? 

I tend to keep things one version behind to give other packages a chance to release compatible version on conda-forge

Reply:
> > Any reason not to go to 3.10?
> 
> I tend to keep things one version behind to give other packages a chance to release compatible version on conda-forge

3.10 is a maintenance release with no breaking changes:
https://www.python.org/downloads/release/python-3910/

Since we had this already at 3.10, my vote is to keep going forward.

Reply:
ok! done in the last commit